financial-templates-lib/ExpiringMultiPartyClient.js

```diff
@@ -121,6 +121,12 @@ class ExpiringMultiPartyClient {
     for (const address of this.sponsorAddresses) {
       const liquidations = await this.emp.methods.getLiquidations(address).call();
       for (const [id, liquidation] of liquidations.entries()) {
+        // Liquidations that have had all of their rewards withdrawn will still show up here but have their properties set to default values.
+        // We can skip them.
+        if (liquidation.state === "0") {
```

Comment:
Nit: Can we use an object to encode this enum to be consistent with other implementations and the tests? makes it easier to read.

Reply:
Let me add this in a follow up PR, so that I can have all the tests reference the same global object.